integration/src/test/resources/snapshot_consumer.properties

```diff
@@ -4,4 +4,5 @@ max.poll.interval.ms=10000
 enable.auto.commit=false
 metadata.max.age.ms=10000
 auto.offset.reset=earliest
-bootstrap.servers=127.0.0.1:9092
\ No newline at end of file
+bootstrap.servers=127.0.0.1:9092
+group.id=drools
```

Comment:
What is the scope of this group.id?

Reply:
If isn't present in the consumer configuration kafka 2.2.1 throws an exception, is to group the consumers, in the 2.1.1 seems not mandatory

Reply:
Can you please verify what was the default value in 2.1.1? Because if previous values was "random" and now is the same for all the consumer we could have a differen behavior

Reply:
ok